sdk/textanalytics/ai-text-analytics/src/models.ts

```diff
@@ -391,7 +443,512 @@ export interface Opinion {
+
+/**
+ * A healthcare entity that plays a specific role in a relation.
+ */
+export interface HealthcareEntityRelationRole {
+  /**
+   * A healthcare entity
+   */
+  entity: HealthcareEntity;
+  /**
+   * The role of the healthcare entity in a particular relation.
+   */
+  name: HealthcareEntityRelationRoleType;
+}
+
+/**
+ * A relationship between two or more healthcare entities.
+ */
+export interface HealthcareEntityRelation {
+  /**
+   * The type of the healthcare relation.
+   */
+  relationType: RelationType;
+  /**
+   * The list of healthcare entities and their roles in the healthcare relation.
+   */
+  roles: HealthcareEntityRelationRole[];
+}
+
+/**
+ * The results of a successful healthcare operation for a single document.
+ */
+export interface HealthcareSuccessResult extends TextAnalysisSuccessResult {
+  /**
+   * Healthcare entities.
+   */
+  entities: HealthcareEntity[];
+  /**
+   * Relations between healthcare entities.
+   */
+  entityRelations: HealthcareEntityRelation[];
+  /**
+   * JSON bundle containing a FHIR compatible object for consumption in other
+   * Healthcare tools. For additional information see {@link https://www.hl7.org/fhir/overview.html}.
+   */
+  fhirBundle?: { [propertyName: string]: any };
+}
+
+/**
+ * An error result from the healthcare operation on a single document.
+ */
+export type HealthcareErrorResult = TextAnalysisErrorResult;
+
+/**
+ * The result of the healthcare operation on a single document.
+ */
+export type HealthcareResult = HealthcareSuccessResult | HealthcareErrorResult;
+
+/**
+ * The result of the custom recognize entities operation on a single document.
+ */
+export type CustomEntityRecognitionResult =
+  | CustomEntityRecognitionSuccessResult
+  | CustomEntityRecognitionErrorResult;
+
+/**
+ * The result of the recognize custom entities operation on a single document,
+ * containing a collection of the entities identified in that document.
+ */
+export interface CustomEntityRecognitionSuccessResult extends TextAnalysisSuccessResult {
+  /**
+   * The collection of entities identified in the input document.
+   */
+  entities: Entity[];
+}
+
+/**
+ * An error result from the recognize custom entities operation on a single document.
+ */
+export type CustomEntityRecognitionErrorResult = TextAnalysisErrorResult;
+
+/**
+ * The result of the custom classify document single category operation on a single document.
+ */
+export type CustomSingleLabelClassificationResult =
+  | CustomSingleLabelClassificationSuccessResult
+  | CustomSingleLabelClassificationErrorResult;
+
+/**
+ * The result of the custom classify document single category operation on a single document,
+ * containing the result of the classification.
+ */
+export interface CustomSingleLabelClassificationSuccessResult extends TextAnalysisSuccessResult {
+  /**
+   * The collection of classifications in the input document.
+   */
+  classifications: ClassificationCategory[];
```

Comment:
the service doesn't represent this as a list in the April version for single label. Are you just doing this now in advance of the merge of single/multi classification in the May version?

Reply:
Yes! I am trying to minimize the future changes since I already know about them.